submarine-server/server-core/src/test/java/org/apache/submarine/server/workbench/database/service/MetricServiceTest.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.submarine.server.workbench.database.service;
+
+import org.apache.submarine.server.workbench.database.entity.Metric;
+import org.junit.After;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.math.BigInteger;
+import java.util.List;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+public class MetricServiceTest {
+  private static final Logger LOG = LoggerFactory.getLogger(MetricServiceTest.class);
+  MetricService metricService = new MetricService();
+
+  @After
+  public void removeAllRecord() throws Exception {
+    List<Metric> metricList = metricService.selectAll();
+    LOG.info("jobList.size():{}", metricList.size());
+    for (Metric metric : metricList) {
+      metricService.deleteById(metric.getId());
+    }
+  }
+
+  @Test
+  public void testSelect() throws Exception {
+    Metric metric = new Metric();
+    metric.setMetricKey("score");
+    metric.setValue((float) 0.666667);
+    metric.setWorkerIndex("worker-1");
+    metric.setTimestamp(new BigInteger("1569139525097"));
+    metric.setStep(0);
+    metric.setIsNan(0);
+    metric.setJobName("application_1234");
+    boolean result = metricService.insert(metric);
+    assertNotEquals(result, -1);
```

Comment:
result is boolean type, It should be compared to `true` or `false`, not to `-1`

assertTrue(result);